setup.py

```diff
@@ -84,7 +84,7 @@
       license='BSD',
       url='http://astropy.org',
       long_description=astropy.__doc__,
-      download_url='%s/astropy-%s.tar.gz' % (DOWNLOAD_BASE_URL, VERSION),
+      download_url='{0}/astropy-{1}.tar.gz'.format(DOWNLOAD_BASE_URL, VERSION),
```

Comment:
@pllim - I know my excuse, I've only looked at the ``astropy`` subdir. But even there, I guess, there are a few more, and a few will slip through review in the future, too.